example_site/src/components/ConfigurationControls.js

```diff
@@ -134,306 +150,348 @@ export default function ConfigurationControls({ isOpen }) {
 
   const formClasses = classnames({
     "configuration-controls-form": true,
-    panel: true,
-    open: isOpen,
-    closed: !isOpen,
+    'modal-fade': true,
+     panel: true,
+     open: isOpen,
+     closed: !isOpen,
+  });
+
+  const titlePanel = classnames({
+    "title-description": true,
+     panel: true,
+     open: isOpen,
+     closed: !isOpen,
   });
 
   return (
-    <form onSubmit={handleSubmit(onSubmit)} className={formClasses}>
-      <div>
-        <h2>Configuration</h2>
+    <form onSubmit={handleSubmit(onSubmit)} className={titlePanel}>
+      <div className='center-title'>
+        <div className='Header'>
```

Comment:
Do you need both divs here? Each element has its own properties and rules and will affect the layout of your page, so it's good practice to only include elements that you need to achieve your layout.

Reply:
No, the div containing the className='Header' was not needed!